### ISSUE TITLE
Add Actions support

### ISSUE THREAD

**Issue by marcolettieri** [mwaylabs/flutter-apns#15]
i'm going to fork and add the actions buttons for the library, have you already considered this feature?

**Comment by dylanmoo** [comment]
This is a requirement for my project as well. How's your progress coming along @marcolettieri ? I've already implemented a solution in swift that could be generalized for use in this package.

**Comment by imtoori** [comment]
@dylanmoo how did you do that? Using a unnotificationserviceextension?

**Comment by dylanmoo** [comment]
@imtoori No extension is necessary. You can use UserNotifications framework to register a set of UNNotificationCategory that contain a sets of UNNotificationAction like so: 
```
var categories: [UNNotificationCategory]?
categories = ...
UNUserNotificationCenter.current().setNotificationCategories(categories)
```

and then implement the delegate methods in UNUserNotificationCenterDelegate

**Comment by szotp** [comment]
Added on master. I will test it after the weekend and release new version.

**Comment by lordgreg** [comment]
The enhancement was released with version [1.4](https://github.com/mwaylabs/flutter-apns/tree/1.4.0).
